### PR TITLE
Refactor/minor changes

### PR DESCRIPTION
**Pull request by UltraCakeBakery**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "./.svelte-kit/tsconfig.json"
```
Comment (justingolden21) [inline tsconfig.json:2]: Why is this necessary?

File: todo.txt [hunk 2]
```diff
@@ -1,9 +0,0 @@
-urgent:
-n/a
-
-future:
-create and implement related articles component
-
-maybe:
-search bar
```
Comment (justingolden21) [inline todo.txt:8]: We still want to add search bar and pagination in the future. We could create a github task

File: src/routes/(rgb)/+page.js [hunk 3]
```diff
@@ -1,7 +1,5 @@
-
 export async function load({ fetch }) {
 	const res = await fetch('/blog/posts.json');
-	const { posts } = await res.json();
-
+	const posts = await res.json();
```
Comment (justingolden21) [inline src/routes/(rgb)/+page.js:3]: Did this not work before? It should either have been broken before or broken now, right? Did we restructure the data in this PR?

Comment (UltraCakeBakery) [reply src/routes/(rgb)/+page.js:3]: If I remember correctly this still worked due to some javascript magic in both cases. I forgot what the explanation was though. It works now, and looking at the json data, this should be correct. There is no `posts` property on the response.. right?

File: todo.txt (hunk 2 above)
Comment (UltraCakeBakery) [reply todo.txt:8]: In the comment of the github commit I said something along the lines of creating issues in github with a `whenever` milestone or something like that.

File: tsconfig.json (hunk 1 above)
Comment (UltraCakeBakery) [reply tsconfig.json:2]: svelte kit automatically generates typehinting for API endpoints and other stuff. This makes those typehints actually work in your IDE
